src/extension/browserAction/previously-enabled.html

```diff
@@ -0,0 +1,26 @@
+<html>
```

Comment:
Is the intent to have this page show up if the user clicks the "I already enabled 2FA" link a second time?

If yes, what do you think about just updating the text on the initial popup page to say "You've already enabled 2FA on website.com" instead of having a link to click at all? One less click.